server/cli/cli.go

```diff
@@ -43,6 +43,9 @@ Output build version information.
 		fmt.Fprintf(w, "Build Time:  %s\n", info.Time)
 		fmt.Fprintf(w, "Build Deps:\n\t%s\n",
 			strings.Replace(strings.Replace(info.Deps, " ", "\n\t", -1), ":", "\t", -1))
+		if err := w.Flush(); err != nil {
+			fmt.Println(err)
```

Comment:
I would expect this error to be something to do with malformed tabular input, not a real failure to flush the buffer. why not just print it?